src/robotide/log/log.py

```diff
@@ -62,7 +65,10 @@ def _remove_old_log_files(self):
     @property
     def _logfile(self):
         if self._outfile is None:
-            self._outfile = open(self._path, 'w', encoding='utf8')
+            if PY2:
+                self._outfile = io.open(self._path, 'w', encoding='utf8')
+            else:
+                self._outfile = open(self._path, 'w', encoding='utf8')
```

Comment:
There is no need for condition. Use io.open for both python 2 and python 3.
On python 3 io.open is the same as open. Only on python 2 it's different.

Reply:
OK.